xe/do.py

```diff
@@ -25,7 +27,8 @@ def withenv(cmd):
 def run(cmd):
     venv = path(settings['VENV'])
     if venv.exists():
-        activate = venv / 'bin' / 'activate_this.py'
+        bin_dir = 'bin' if not os.name == 'nt' else 'Scripts'
```

Comment:
Can we move this to a function called `bin_dir` that is basically this line? It can then be used in the `env_do` function as well.